nilearn/datasets/tests/test_neurovault.py

```diff
@@ -0,0 +1,830 @@
+"""
+Test the neurovault module.
+"""
+# Author: Jerome Dockes
+# License: simplified BSD
+
+import os
+import warnings
+import stat
+try:
+    from os.path import samefile
+except ImportError:
+    # os.path.samefile not available on windows
+    samefile = None
+import tempfile
+import shutil
+import json
+import re
+
+import numpy as np
+from nose import SkipTest
+from nose.tools import (assert_true, assert_false, assert_equal, assert_raises)
+try:
+    from nose.tools import assert_warns
+except ImportError:
+    assert_warns = None
```

Comment:
You should use the backport in scikit-learn: sklearn.utils.testing.assert_warns.